### ISSUE TITLE
Use paths from metadata instead of hardcoded into SK displays

### ISSUE THREAD

**Issue by colemanw** [graydigitalgroup/com.graydigitalgroup.committees#3]
I assume this was for testing purposes, but 75df17b2c213feac82320476e91b19691a79a8ba removed some valuable metadata.

Once you get your paths stabilized, I recommend you add them back to the metadata and switch your SK display links to use `entity+action` instead of `path`. The reason it's better is because SK will check permissions in the former, and hide links that the user does not have permission to use (e.g. if they do not have permission to edit or delete committees, then SK will hide that link). With path-based links, SK doesn't know anything about it so you don't get that benefit.

**Comment by rbaugh** [comment]
The paths were technically going to the FB routes as I was using FB forms instead of the older Forms in the ext.

**Comment by colemanw** [comment]
Yes but that's ok. If you update the paths in your schema xml files to match the FB routes, then everything will work, and SK will be able to automatically link to those forms.
